workflow/rules/common.smk

```diff
@@ -299,8 +310,13 @@ def get_stratified_truth(suffix=""):
 
 
 def get_confidence_regions(wildcards):
```

Comment:
Target and confidence regions probably do not need to be modified both.

I would use the confidence regions.